internal/cypress/config.go

```diff
@@ -54,6 +54,12 @@ type Cypress struct {
 	// Version represents the cypress framework version.
 	Version string `yaml:"version" json:"version"`
 
+	// Record represents the cypress framework record flag.
+	Record bool `yaml:"record" json:"record"`
+
+	// Key represents the cypress framework key flag.
+	Key string `yaml:"key" json:"key"`
```

Comment:
I'm not sur it make sense to have a dedicated flag for the cypress key (if this stands for cypress record key).
It can be provided by `CYPRESS_RECORD_KEY` env variable (and it also suggests to write a secret in the code source, which should be avoided.

Reply:
@FriggaHel I don't think the cypress record key is a secret. It's just a unique identifier for a Cypress parallel session. It's the API key that is secret.

Reply:
True though, we could potentially leave out the `Key` attribute.
Both are fine imho, since it aligns with what cypress supports (i.e. it accepts both an env variable and a CLI flag).

Reply:
@dpgraham they consider it as a secret, and suggests to avoid exposure (in case of public repo): https://docs.cypress.io/guides/core-concepts/dashboard-service.html#Authentication

We can keep it, but we should suggest to use the env-var method :)

Reply:
> We can keep it, but we should suggest to use the env-var method :)
Why? It makes no difference to us in terms of how it's going to be processed. Both will end up being serialized.

Having said that, @valentyn88 , please keep in mind that this value needs to be expanded (e.g. at the time the config is read in saucectl), because we wouldn't want the customer to specify the key via the config directly, but rather as an env var.

This ought to be possible:
```
cypress:
  key: $MY_SECRET_KEY
```

Reply:
@alexplischke agreed for serialisation, my concern was mainly because env vars was not expanded here.

> ```
> cypress:
>   key: $MY_SECRET_KEY
> ```
=> LGTM if we can set it by env var.